api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -96,6 +116,8 @@ message DownstreamTlsContext {
     lt {seconds: 4294967296}
     gte {}
   }];
+
+  OcspStaplePolicy ocsp_staple_policy = 8 [(validate.rules).enum = {defined_only: true}];
```

Comment:
Probably want a doc line for this with a note that it defaults to SKIP_STAPLING_IF_EXPIRED.